### PR TITLE
Uploadsong

### PR DESCRIPTION
**Pull request by YogiHa**
Hi :)

have completed both criteria and ready for a PR.

pls comment with your insights and review :)

### CODE REVIEW

File: src/components/PlayerBar/UploadButton/UploadButton.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+// ToDo - create some desigen to the button
+
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { clearSong } from '../../../store/actions/songActions';
```
Comment (YogiHa) [inline src/components/PlayerBar/UploadButton/UploadButton.js:5]: by calling this action form the store,
state.song === null
and AppBrowser.js should send the user back to upload song mockup

File: src/components/UploadSong/UploadSong.jsx [hunk 2]
```diff
@@ -1,51 +1,94 @@
...
+                    </svg>
+                </button>
+                <div className={classes.uploadText}>Upload New Song</div>
+                <div className={classes.fileFormatStyle}>
+                    file format supported: mp3
                 </div>
             </div>
-        );
-    }
+
```
Comment (YogiHa) [inline src/components/UploadSong/UploadSong.jsx:78]: here the button with hidden style

**Review by ArnasDickus — COMMENTED** [review]
Do you have account I can use it? Because currently, I can't access app.js without logging in.

**Comment by YogiHa** [comment]
you can log with my a@a.com fake user 
or register your b@b.com fake one

**Comment by ArnasDickus** [comment]
@YogiHa So I'm not following you. From my understanding login is working. In order to login, you need to:
1) Add config file configuration to the project [trello task](https://trello.com/c/6AtrDWoA/72-implement-firebase)
2) Try to login, a@a.com, however, I get error.

problems with login

**Comment by YogiHa** [comment]
answered you at discord

**Review by ArnasDickus — APPROVED** [review]
Alright, finally I looked at it, and have no issues.

**Pull request merged**